### PR TITLE
Add options to convert_model function

### PR DESCRIPTION
**Pull request by Svan70**
I needed to set score_threshold and max_detections for the inference model.
Given the current code It is straightforward and could be useful, that's why I 
submit this PR. It just add the possibility to set these parameters in convert_model function (and in convert_model script to be consistent).

By the way, many thanks for this package.

### CODE REVIEW

File: keras_retinanet/bin/convert_model.py [hunk 1]
```diff
@@ -43,6 +43,10 @@ def parse_args(args):
     parser.add_argument('--no-nms', help='Disables non maximum suppression.', dest='nms', action='store_false')
     parser.add_argument('--no-class-specific-filter', help='Disables class specific filtering.', dest='class_specific_filter', action='store_false')
     parser.add_argument('--config', help='Path to a configuration parameters .ini file.')
+    parser.add_argument('--nms-threshold', help='Value for non maximum suppression threshold', dest='nms_threshold', type=float, default=0.5)
```
Comment (hgaiser) [inline keras_retinanet/bin/convert_model.py:46]: No need to define `dest`, these are already the defaults.

Comment (hgaiser) [reply keras_retinanet/bin/convert_model.py:46]: Bit nitpicky, but could you add a period after the `help` sentence?

File: keras_retinanet/bin/convert_model.py [hunk 2]
```diff
@@ -74,7 +78,9 @@ def main(args=None):
     models.check_training_model(model)
 
     # convert the model
-    model = models.convert_model(model, nms=args.nms, class_specific_filter=args.class_specific_filter, anchor_params=anchor_parameters)
+    model = models.convert_model(model, nms=args.nms, class_specific_filter=args.class_specific_filter, anchor_params=anchor_parameters,
```
Comment (hgaiser) [inline keras_retinanet/bin/convert_model.py:81]: Also nitpicky, but our style for long lines is:
```python
foo(
    bar,
    baz=1,
    foobar=2
)
```
It's more lines, but IMO more readable.

File: keras_retinanet/models/__init__.py [hunk 3]
```diff
@@ -87,14 +87,15 @@ def load_model(filepath, backbone_name='resnet50'):
...
-def convert_model(model, nms=True, class_specific_filter=True, anchor_params=None):
+def convert_model(model, nms=True, class_specific_filter=True, anchor_params=None, **kwargs):
     """ Converts a training model to an inference model.
 
     Args
         model                 : A retinanet training model.
         nms                   : Boolean, whether to add NMS filtering to the converted model.
         class_specific_filter : Whether to use class specific filtering or filter for the best scoring class only.
         anchor_params         : Anchor parameters object. If omitted, default values are used.
+        **kwargs              : Inference and minimal retinanet model settings
```
Comment (hgaiser) [inline keras_retinanet/models/__init__.py:98]: Similarly here, period at the end.

File: keras_retinanet/bin/convert_model.py (hunk 1 above)
Comment (Svan70) [reply keras_retinanet/bin/convert_model.py]: Done

File: keras_retinanet/bin/convert_model.py (hunk 2 above)
Comment (Svan70) [reply keras_retinanet/bin/convert_model.py]: Done

File: keras_retinanet/models/__init__.py (hunk 3 above)
Comment (Svan70) [reply keras_retinanet/models/__init__.py]: Done

**Comment by Svan70** [comment]
Are there any unitary / integrated tests I have to run / write ?

File: keras_retinanet/bin/convert_model.py (hunk 2 above)
Comment (hgaiser) [reply keras_retinanet/bin/convert_model.py]: Not exactly what I had in mind, could you have another look at this?

Comment (Svan70) [reply keras_retinanet/bin/convert_model.py]: Ok. I did it similarily to your other function calls. Like this ?

**Comment by hgaiser** [comment]
> Are there any unitary / integrated tests I have to run / write ?

Normally I'd say yes, but in this case it is just argument passing. Thanks again for the PR!

**Pull request merged**